pkg/apis/core/v1alpha1/types_backupbucket.go

```diff
@@ -82,3 +82,8 @@ type BackupBucketProvider struct {
 	// Region is the region of the bucket.
 	Region string `json:"region"`
 }
+
+const (
+	// BackupBucketEventReferenced indicates that the BackupBucket is still being referenced by at least one BackupEntry
+	BackupBucketEventReferenced = "Referenced"
```

Comment:
Maybe its more convenient to define a constant called `ResourceReferenced` (could be in pkg/apis/core/v1alpha1/constants/types_constants.go)  or so and reuse it in all the required places since it will always have the same value?

Reply:
Done, PTAL :)